pxr/usd/usd/testenv/testCoreUsdValidators.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2024 Pixar
+//
+// Licensed under the terms set forth in the LICENSE.txt file available at
+// https://openusd.org/license.
+//
+
+#include "pxr/usd/usd/validator.h"
+#include "pxr/usd/usd/validationError.h"
+#include "pxr/usd/usd/validatorTokens.h"
+#include "pxr/usd/usd/validationRegistry.h"
+
+#include <iostream>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+static
+void TestCoreUsdStageMetadata()
+{
+
+    // Get stageMetadataChecker
+    UsdValidationRegistry &registry = UsdValidationRegistry::GetInstance();
+    const UsdValidator *validator = registry.GetOrLoadValidatorByName(
+            UsdValidatorNameTokens->stageMetadataChecker);
+    TF_AXIOM(validator);
+
+    // Create an empty stage
+    SdfLayerRefPtr rootLayer = SdfLayer::CreateAnonymous();
+    UsdStageRefPtr usdStage = UsdStage::Open(rootLayer);
```

Comment:
You could have directly done UsdStage::CreateInMemory here, and then you do not need to use the SdfLayer at all.

Reply:
I see, yeah makes sense. Sorry I missed the check on line 40.